examples/grpc-client-streaming/grpc_client_streaming_service.bal

```diff
@@ -1,33 +1,54 @@
 // This is the server implementation for the client streaming scenario.
 import ballerina/grpc;
-import ballerina/log;
+import ballerina/io;
+
+listener grpc:Listener ep = new (9090);
 
 @grpc:ServiceDescriptor {
     descriptor: ROOT_DESCRIPTOR,
     descMap: getDescriptorMap()
 }
-service /HelloWorld on new grpc:Listener(9090) {
-
-    isolated function lotsOfGreetings(grpc:Caller caller,
-                            stream<string,error> clientStream) {
-        log:print("Client connected successfully.");
-        //Read and process each message in the client stream
+service "HelloWorld" on ep {
+    remote function lotsOfGreetings(stream<string,error> clientStream) returns string|error {
+        io:println("connected sucessfully.");
```

Comment:
Aren't we supposed to use `log` in services and `io` in clients?

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/1180/commits/05e21f2d92ca432dc94ef290c343eb509c05ef7e